MediaBrowser.Controller/Lyrics/Metadata.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// Metadata model.
+/// </summary>
+public class Metadata
+{
+    /// <summary>
+    /// Gets or sets Artist - [ar:The song artist].
+    /// </summary>
+    public string? Ar { get; set; }
```

Comment:
Can these not have more descriptive names?

Reply:
Possibly not unless the mapper can use different variable names similar to XML or JSON parsers.

Reply:
We shouldn't use a mapper and yes it can